src/brevitas/graph/equalize.py

```diff
@@ -145,11 +145,16 @@ def _cross_layer_equalization(srcs: List[nn.Module], sinks: List[nn.Module]):
 
     src_axes = {m: _get_output_axis(m) for m in srcs}
     sink_axes = {m: _get_input_axis(m) for m in sinks}
-    src_size = _get_size(src_axes)
-    sink_size = _get_size(sink_axes)
+    try:
+        src_size = _get_size(src_axes)
+        sink_size = _get_size(sink_axes)
+    except RuntimeError:
```

Comment:
RuntimeError is not supposed to be internally catched like this. We should define an an exception, and then catch that.